source/extensions/filters/http/rbac/rbac_filter.cc

```diff
@@ -109,13 +109,15 @@ RoleBasedAccessControlFilter::decodeHeaders(Http::RequestHeaderMap& headers, boo
   const auto engine =
       config_->engine(callbacks_->route(), Filters::Common::RBAC::EnforcementMode::Enforced);
   if (engine != nullptr) {
+    std::string effective_policy_id = "none";
```

Comment:
Erm this is a behavior change and it will be put into request metadata. I'd like to keep this empty, but you can still quote it in debug log so you know it is?

Reply:
@lizan fixed this one, 

As c++ doesn't have the null coalescing operator (??) I used the ternary operator to add "none" if effective_policy_id is empty